README.md

```diff
@@ -1,2 +1,49 @@
-# rfcs
-RFCs for additions or changes to MOXY open-source projects
+# MOXY OSS RFCs
+
+This repository is intended to be used for reviewing submitted RFCs related to additions or changes to MOXY OSS projects and processes.
```

Comment:
```suggestion
This repository is intended to be used for reviewing submitted RFCs (request for comments) related to additions or changes to MOXY OSS projects and processes.
```
I think it is better to present the initials in the first occurrence in the text